modules/rostests/apitests/kernel32/precomp.h

```diff
@@ -11,5 +11,7 @@
 #include <winreg.h>
 #include <strsafe.h>
 #include <ndk/umtypes.h>
+#include <ndk/extypes.h>
+#include <ndk/ntndk.h>
```

Comment:
@binarymaster : to determine which other ndk headers you need instead of the generic ntndk.h, first comment that one out, then recompile, look at the missing identifier names and search where they are defined.